multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/IncrementalAppInstancesUpdateStep.java

```diff
@@ -188,7 +188,10 @@ protected String getStepErrorMessage(ProcessContext context) {
     public Duration getTimeout(ProcessContext context) {
         CloudApplicationExtended application = context.getVariable(Variables.APP_TO_PROCESS);
         Duration timeout = calculateTimeout(context, TimeoutType.START);
-        return Duration.ofSeconds(Math.min(timeout.getSeconds() * application.getInstances(), MAX_TIMEOUT));
+        int minRequiredInstances = application.getInstances() <= 0 ? 1 : application.getInstances();
```

Comment:
I personally think that `Math.max(application.getInstances(), 1);` looks cleaner here

Reply:
done